config/locales/overrides/traininghub.en.yml

```diff
@@ -562,10 +562,10 @@ en:
       TeSS has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement No. <a href="https://cordis.europa.eu/project/rcn/198519_en.html", target="_blank">676559</a>.
   guides:
     webinar:
-      name: 'Asynchronous Webinar Development Guide'
-      type: 'creation guide'
-      image: 'scilifelab/guides/asynchpreview.png'
```

Comment:
are we using the asynchpreview.png image somewhere else? If not we can delete it from git

Reply:
this image is used as a placeholder for the video training manual currently, but will not appear on the website once this is updated